EcpBtBot/DialogFlow/Response.py

```diff
@@ -19,7 +19,7 @@ def status(self):
 class Result:
```

Comment:
Also add a pydoc (better 3 ") to explain what the class does. but good to know you do a class :)